core/runtime/bindings.rs

```diff
@@ -801,7 +801,7 @@ pub fn create_exports_for_ops_virtual_module<'s>(
   op_ctxs: &[OpCtx],
   scope: &mut v8::HandleScope<'s>,
   global: v8::Local<v8::Object>,
-) -> Vec<(FastString, v8::Local<'s, v8::Value>)> {
+) -> Vec<(FastStaticString, v8::Local<'s, v8::Value>)> {
```

Comment:
Do you think it would make sense to add `FastConstString`? Eg for virtual modules we always now upfront the names and can make all the names const strings.